hack/make/tgz

```diff
@@ -9,6 +9,11 @@ if [ ! -d "$CROSS/linux/amd64" ]; then
 	false
 fi
 
+bundle .integration-daemon-start
+bundle .detect-daemon-osarch
+
+make manpages
```

Comment:
This is currently the way to do it. We could probably improve the Makefile and not use a .PHONY target, which might allow it to skip the step if they exist already (and haven't been modified).

I don't think a "skip if exists" check is reliable, they could be out of date.

On a slightly related note, I've been hacking on [dobi](https://github.com/dnephin/dobi) which is designed to handle exactly this type of problem. Run build tasks in containers, but skip it if they are still fresh (and only ever run a task once per "chain"). I think it's featureful enough at this point that I might be able to open a PoC PR against docker/docker.